packages/ng-table-builder/src/table-builder.component.html

```diff
@@ -51,16 +54,13 @@
         (observeVisible)="calculateViewport(true)"
         [class.table-grid__native-scrollbar]="nativeScrollbar"
         [class.table-grid__scroll-offset]="viewPortInfo?.scrollTop! > 0"
+        [style.height]="'calc(100% - ' + (headerClientHeight + footerClientHeight) + 'px)'"
```

Comment:
what do you think about create pipe? (with test for pipe)

Reply:
fix. but it can't be pure